content/en/community/roadmap.md

```diff
@@ -0,0 +1,19 @@
+# CHT Roadmap
```

Comment:
```suggestion
---
title: "CHT Roadmap"
linkTitle: "Roadmap"
weight: 3
description: >
  CHT Initiatives
---
```

Reply:
Currently, the page doesn't appear in the sidebar, as the page configuration is missing.